test/services/user/services/userService.test.js

```diff
@@ -9,8 +10,7 @@ const coursesService = app.service('courses');
 const testObjects = require('../../helpers/testObjects')(app);
 const { equal: equalIds } = require('../../../../src/helper/compare').ObjectId;
 
-let testUserId;
-const testGenericErrorMessage = 'Der angefragte Nutzer ist unbekannt!';
+const usersFromDifferentSchoolsErrorMessage = 'Der angefragte Nutzer gehört nicht zur eigenen Schule!';
```

Comment:
We dont want to give the user information that the user exists on a different school, if there is no such user on the users school the error message should be the same no matter if he exists in a different school or not.

It would be really good if you could revert this change, and make sure to adjust the user services accordingly.

Reply:
:+1: